kernel/kernel-impl/src/main/resources/antisamy/high-security-policy.xml

```diff
@@ -22,7 +22,7 @@
     <!-- FORMATTING -->
     <directive name="formatOutput" value="false" /><!-- default: true -->
     <directive name="preserveComments" value="false" /><!-- default: false -->
-    <directive name="preserveSpace" value="false" /><!-- default: false -->
+    <directive name="preserveSpace" value="true" /><!-- default: false -->
```

Comment:
Has this been changed in the past? If yes, why was it previously changed?

Reply:
I don't see preserveSpace ever changing. I changed formatOutput in KNL-1261.

Setting this to true sounds okay to me.

Reply:
Ahh, thanks. I was worried that this was reverting a previous change.